code/controllers/subsystem/pai.dm

```diff
@@ -195,4 +217,5 @@ SUBSYSTEM_DEF(pai)
 	var/description
 	var/role
 	var/comments
+	var/color = "#2DDF1D"
```

Comment:
Does this value need to match up with the global list in some way?

Reply:
As far as I am aware, no?

Reply:
Yes, that would be Data Green, which honestly we might want to make into defines for this